test/ci/buildkite/setup.yml

```diff
@@ -1,3 +1,6 @@
+env:
+  NGRAPH_TF_BACKEND: ${NGRAPH_TF_BACKEND:-CPU}
```

Comment:
Do we need a global setting?

Reply:
Yes, this setting is needed to set the backend correctly for all steps in case of each pipeline.